### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by maddie**

### CODE REVIEW

**Comment by maddie** [comment]
This commit aims to improve the grammar and editability of the README

File: README.md [hunk 1]
```diff
@@ -78,36 +84,38 @@ Configuration file is "config.json" by default:
...
+            + auto: If client IP is not in the reserved IP network, use client IP. Otherwise, use external IP.
             + disable: Disable this feature.
-        + ExternalIP: If this field is empty, edns client subnet will be disabled when use it.
-+ RedirectIPv6Record: Redirect IPv6 dns query to alternative dns.
-+ File: For windows user, if you want to use absolute path, please try like this: "C:\\\xx\\\xx.txt".
-+ DomainBase64Decode: If this file is base64 decoded, use "true".
-+ MinimumTTL: Set the minimum TTL value (second) in order to improve cache sufficiency, use 0 to disable.
+        + ExternalIP: If this field is empty, EDNS client subnet will be disabled when used.
++ RedirectIPv6Record: Redirect IPv6 DNS queries to alternative DNS servers.
++ File: For Windows user, if you want to use absolute path, please escape backslashes properly, e.g. `C:\\\xx\\\xx.txt`.
```
Comment (holyshawn) [inline README.md]: It should be "C:\\xx\\xx.txt" in json file, and it will be interpret as "C:\xx\xx.txt" in Go.

Comment (maddie) [inline README.md]: So there's no need to have it escaped? Then I'll rephrase this whole sentence.

Comment (holyshawn) [inline README.md]: Markdown once, json twice, and the result will be "X:\path\to\xx.txt".

Comment (holyshawn) [inline README.md]: Maybe it is the "\``" problem, when you are using "\``",  there is no escape character inside.

**Comment by maddie** [comment]
The path example for Windows users should be OK now.

File: README.md [hunk 2]
```diff
@@ -78,36 +84,39 @@ Configuration file is "config.json" by default:
...
+            + auto: If client IP is not in the reserved IP network, use client IP. Otherwise, use external IP.
             + disable: Disable this feature.
-        + ExternalIP: If this field is empty, edns client subnet will be disabled when use it.
-+ RedirectIPv6Record: Redirect IPv6 dns query to alternative dns.
-+ File: For windows user, if you want to use absolute path, please try like this: "C:\\\xx\\\xx.txt".
-+ DomainBase64Decode: If this file is base64 decoded, use "true".
-+ MinimumTTL: Set the minimum TTL value (second) in order to improve cache sufficiency, use 0 to disable.
+        + ExternalIP: If this field is empty, EDNS client subnet will be disabled when used.
++ RedirectIPv6Record: Redirect IPv6 DNS queries to alternative DNS servers.
++ File: For Windows users, you can use relative path like `.\file.txt`, or absolute path like `C:\path\to\file.txt` in
```
Comment (holyshawn) [inline README.md]: It should be written as `C:\\path\\to\\file.txt`in json file. Markdown has my word escaped.........

Comment (holyshawn) [inline README.md]: And relative path should be `./file.txt`

**Comment by holyshawn** [comment]
Merged. Good practice, thank you.

**Pull request merged**